packages/taxdat/R/postprocessing.R

```diff
@@ -0,0 +1,712 @@
+                  pop = output_location_pop$mean) %>% 
+    dplyr::bind_cols(get_output_sf_reaload(config_list = config_list,
+                                           redo = redo_aux), .)
+  
+  risk_cat
+}
+
+#' postprocess_pop_at_risk
+#'
+#' @param config_list 
+#' @param redo_aux 
+#'
+#' @return
+#' @export
+#'
+#' @examples
+postprocess_pop_at_risk <- function(config_list,
+                                    redo_aux = FALSE) {
+  
+  # Get genquant data
+  genquant <- readRDS(config_list$file_names$stan_genquant_filename) 
+  
+  # Get dictionnary of risk categories
+  risk_cat_dict <- get_risk_cat_dict()
+  
+  # Get mean annual incidence summary
+  pop_at_risk <- genquant$summary("tot_pop_risk") %>% 
+    dplyr::mutate(risk_cat = risk_cat_dict[as.numeric(str_extract(variable, "(?<=\\[)[0-9]+(?=,)"))],
+                  risk_cat = factor(risk_cat, levels = risk_cat_dict),
+                  admin_level = as.numeric(str_extract(variable, "(?<=,)[0-9]+(?=\\])")) - 1,
+                  country = get_country_from_filename(config_list$file_names$stan_genquant_filename))
+  
+  pop_at_risk
+}
+
+
+#' postprocess_mean_annual_incidence
+#' 
+#' @param config_list config list
+#'
+#' @return
+#' @export
+#'
+postprocess_grid_mai_rates <- function(config_list,
+                                       redo_aux = FALSE) {
+  
+  # Get genquant data
+  genquant <- readRDS(config_list$file_names$stan_genquant_filename) 
+  
+  # Get mean annual incidence summary
+  mai_summary <- genquant$summary("space_grid_rates")
+  
+  # Get the output shapefiles and join
+  res <- get_space_grid(config_list = config_list,
+                        redo = redo_aux) %>% 
+    dplyr::bind_cols(mai_summary) %>% 
+    dplyr::select(-variable, -median, -sd, -mad)
+  
+  res
+}
+
+
+#' postprocess_mean_annual_incidence
+#' 
+#' @param config_list config list
+#'
+#' @return
+#' @export
+#'
+postprocess_grid_mai_cases <- function(config_list,
+                                       redo_aux = FALSE) {
+  
+  # Get genquant data
+  genquant <- readRDS(config_list$file_names$stan_genquant_filename) 
+  
+  # Get mean annual incidence rates at space grid level
+  mai_summary <- genquant$summary("space_grid_rates")
+  
+  # Get population and average over space grid
+  mean_pop_sf <- get_mean_pop_grid(config_list = config_list,
+                                   redo = redo_aux)
+  
+  # Get the output shapefiles and join
+  res <- mean_pop_sf %>% 
+    dplyr::bind_cols(mai_summary) %>% 
+    dplyr::select(-variable, -median, -sd, -mad) %>% 
+    dplyr::mutate(dplyr::across(.cols = c("mean", "q5", "q95"),
+                                ~ . * pop))
+  
+  res
+}
+
+
+# Output shapefiles -------------------------------------------------------
+
+#' get_output_sf_wrapper
+#' This function enables loaded pre-extracted output shapefiles
+#'
+#' @param config 
+#' @param data_dir 
+#'
+#' @return
+#' @export
+#'
+#' @examples
+get_output_sf_reaload <- function(config_list,
```

Comment:
**reload** is misspelled in the function name

Reply:
done